tudatBundle/pagmo2/include/pagmo/algorithms/giACOmo.hpp

```diff
@@ -277,32 +286,44 @@ class gi_aco_mo
                 }
 
                 if (gen==1) {
-                    //here you have to initialize the solution archive (SA)
+
+                    std::vector<std::vector<double>> *Sol_Arch=&SA;
```

Comment:
why a pointer here? In pagmo we do not use of pointers if not necessary. What are you trying to achieve here?

Reply:
I indeed forgot to remove it.. I did it first because I defined the solution archive inside the if, but then I moved it outside and I should have removed the pointer. Thank you!